pkg/placement/workload-projector.go

```diff
@@ -698,7 +681,7 @@ func (wp *workloadProjector) syncConfigObject(ctx context.Context, scRef syncerC
 		return true
 	}
 	logger = logger.WithValues("destination", sp)
-	syncfg, err := wp.syncfgClusterLister.Cluster(scRef.Cluster).Get(string(scRef.Name))
+	syncfg, err := wp.edgeClusterClientset.Cluster(scRef.Cluster.Path()).EdgeV2alpha1().SyncerConfigs().Get(wp.ctx, string(scRef.Name), metav1.GetOptions{})
```

Comment:
I mean, why not fetch the SyncerConfig object from the local cache of an informer watching SyncerConfig objects in the KCS?

Reply:
This seems to have only performance impact, it will take time to change+test and currently the code is working. Opened an issue to re-visit it. 
https://github.com/kubestellar/kubestellar/issues/1329